base/CMakeLists.txt

```diff
@@ -152,16 +152,47 @@ find_file(JACKSON2_JAXB_ANNOTATIONS_JAR
 )
 
 execute_process(
-    COMMAND xmvn-resolve jakarta.xml.bind:jakarta.xml.bind-api:4
-    OUTPUT_VARIABLE JAXB_API_JAR
+    COMMAND awk -F= "$1==\"ID\" { print $2 ;}" /etc/os-release
+    OUTPUT_VARIABLE DISTRO
     OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 
-execute_process(
-    COMMAND xmvn-resolve jakarta.activation:jakarta.activation-api:2
-    OUTPUT_VARIABLE JAVAX_ACTIVATION_JAR
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-)
+if(DISTRO STREQUAL "fedora")
```

Comment:
I think we need to do this for RHEL and CentOS as well.

Reply:
Another option, instead of checking for specific OSes, we can check whether `xmvn-resolve` is available.

Reply:
Ack, I shall include.

Reply:
> Another option, instead of checking for specific OSes, we can check whether `xmvn-resolve` is available.

That would be cleaner, but is more "scripty", so I guess there is a trade off to make.